### DISCUSSION TITLE
Setting `score_cutoff` parameter with function `process.extractOne`  causes TypeError encountered when using v1.8.3

### DISCUSSION THREAD

**Discussion by JayPalm** [rapidfuzz/RapidFuzz#168] [Q&A] [answered]
Error encountered using v1.8.3:
```python
File "~/.pyenv/versions/3.10.0/envs/cld/lib/python3.10/site-packages/make_pptx.py", line 242, in make_pptx
    slide_title_analysis_match = process.extractOne(
  File "cpp_process.pyx", line 475, in cpp_process.extractOne
  File "cpp_common.pxd", line 118, in cpp_common.KwargsInit
TypeError: ('Got unexpected keyword arguments: ', 'score_cuttoff')
```

This is the code used in my script:
```python
slide_title_analysis_match = process.extractOne(
            slide_title, # String
            analysis_groups, # List of strings
            scorer=fuzz.partial_token_ratio,
            score_cuttoff=90,
        )
```

This same code does not cause this error with v1.8.2 or v1.5.0 (only two versions I tested with).

**Comment by maxbachmann** [comment, marked as answer]
I made the argument checking stricter in v1.8.3 to prevent errors like yours. You misspelled `score_cutoff` as `score_cuttoff` which was previously silently ignored.

**Comment by JayPalm** [comment]
> You misspelled score_cutoff as score_cuttoff which was previously silently ignored.
How embarrassing... thanks for catching. Everything working fine now, appreciate it!
